templates/vue/src/components/Lightbox/media/MultiContentMedia/PageMenu/index.vue

```diff
@@ -217,103 +235,165 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+.page-nav-toggle {
+  background-color: transparent;
+  padding: 0;
+  position: absolute;
+  left: 20px;
+  top: 20px;
+  z-index: 11;
+
+  &.fullscreen {
+    top: 40px;
+    @media screen and (min-width: 801px) {
+      display: none;
+    }
+  }
+}
+
 .page-nav-wrapper {
+  position: relative;
+  z-index: 0;
+  min-width: 200px;
+
+  &.lightbox {
+    position: absolute;
+    left: 0;
+    top: 0;
+    z-index: 11;
+    border-radius: 15px 0 0 15px;
+  }
+
+  &.fullscreen {
+    height: 100vh;
+    margin: -24px 24px 0 -24px;
+    z-index: 11;
+
+    .page-nav {
+      padding: 4rem 1.5rem 3rem 1.5rem;
+    }
+  }
+
+  &.unit-child {
+    width: 250px;
+    max-width: 25vw;
+  }
+
   .page-nav {
-    position: relative;
-    color: white;
+    color: #ffffff;
     background: #5d656c;
     padding: 2.2rem 1.5rem;
-    transform: translateY(0);
-    transition: all 0.2s ease-in-out;
     font-size: 14px;
     text-align: left;
-    z-index: 0;
     overflow-y: auto;
-    min-width: 200px;
     display: flex;
     flex-direction: column;
 
-    &.lightbox {
-      position: absolute;
-      left: 0;
-      top: 0;
-      z-index: 11;
-      border-radius: 15px 0 0 15px;
+    @media screen and (min-width: 960px) {
+      font-size: clamp(14px, 1.3vw, 16px);
+    }
 
-      &.closed {
-        background: transparent;
-        min-width: 20px;
-        max-width: 20px;
-      }
+    .page-nav-container {
+      text-align: left;
     }
 
-    &.fullscreen {
-      height: "100vh";
-      margin: -24px 24px 0 -24px;
-      z-index: 11;
+    .page-nav-content {
+      &:first-child {
+        margin-top: 9em;
+      }
 
-      &.closed {
-        @media screen and (max-width: 800px) {
-          min-width: calc(16.33px + 3rem);
-          width: calc(16.33px + 3rem);
-        }
+      .page-menu-items {
+        margin-left: 2em;
+        margin-right: -0.5em;
       }
     }
 
-    &.is-unit-child {
-      width: 250px;
-      max-width: 25vw;
+    .logout-link {
+      color: #ffffff;
     }
+  }
+}
 
-    @media screen and (min-width: 960px) {
-      font-size: calc(14px + (2 * (100vw - 960px) / 1280px - 960px));
+$slide-fade-speed: 0.3s;
+
+.slide-fade-enter-active {
+  white-space: nowrap;

```

Comment:
I think it may be better to remove this line (and the one in `.slide-fade-leave-active`). When the node title is too long to fit on the side menu, I am encountering a small stutter in the animation when the text re-wraps at the end of the animation.

With this line:


https://user-images.githubusercontent.com/44929104/177886742-3bb7eae9-7f48-41f0-befa-3e969eb86331.mov


Without this line:

https://user-images.githubusercontent.com/44929104/177886723-c3b3c480-5b3f-471e-ae13-0d6996358165.mov

Reply:
On mine it does this thing while transitioning if without that line:

<img width="288" alt="image" src="https://user-images.githubusercontent.com/12668055/177902072-a1892396-ee30-4ccf-89c3-7925a585a73b.png">

The text "Normal Unit" and "Body A" all fit in one line when opened, but since we're changing the width, they would not fit while transitioning. As to the situation you described, that is indeed problematic but it is a result of fixing this. I'm not sure how we can have both problems fixed....